CenterForOpenScience/ember-osf-web
# [REG-37][Registries] Use bs-popover to display social icons

## Purpose

Add social `sharing-icons` component to registries

## Summary of Changes

- Add new component

## Side Effects

## Feature Flags

## QA Notes

- on Registries discover page, please make sure the popover shows and is centered
on left. Check responsiveness, cross some other browser (I tested in chrome while developing).
- Analytics: please check that click on the icons hits analytics (if this is applicable, not sure).

## Ticket


[[[REG-37]]](https://openscience.atlassian.net/browse/REG-37)

# Reviewer Checklist
